Parsers/Generate dev shortlink.js

```diff
@@ -0,0 +1,17 @@
+/*
+activation_example:!devlink utah
+regex:!devlink
+flags:gmi
+*/
+
+(function(){
+    
+  var term = current.text.replace('!devlink', '').trim();
+  var strMessage = '';
+  if(term) strMessage = 'Here is the short link to '+term+ '`https://devlink.sn/'+term+'`, Have a good day!';
```

Comment:
I think you need a space in the string with your URL before the ` so your link doesn't end up as
>Here is your short link to utah`https://devlink.sn/utah`

`if(term) strMessage = 'Here is the short link to '+term+ ' `https://devlink.sn/'+term+'`, Have a good day!';